othello_gui.py

```diff
@@ -185,4 +185,4 @@ def gameover(self, game, last_move):
     
     game2.play(othello.game(),
                game2.player(lambda x: minimax.alphabeta(x, 4, othello.edge_eval)),
-               player(), True)
+               game2.player(lambda x:minimax.alphabeta(x,4,othello.bench_eval)), True)
```

Comment:
This is GUI file. I would prefer to keep this player testing in game2.py main file..